lib/signalfx.js

```diff
@@ -0,0 +1,334 @@
+'use strict';
+// Copyright (C) 2015 SignalFx, Inc. All rights reserved.
+
+(function () {
+
+  var request = require('request'); // Use for create post request
+  var async = require('async'); // Use fo make post request in separate thread
+  var winston = require('winston'); // Logging library
+
+// Application version and name for User-Agent
+  var name = 'signalfx-nodejs-client';
+  var version = '0.1.0';
```

Comment:
You can derive this value directly from the package.json using `var version = require('../package.json').version;`